includes/class-mailer.php

```diff
@@ -123,15 +126,60 @@ public static function new_follower( $notification ) {
 			$email = $user->user_email;
 		}
 
-		/* translators: %1$s: Blog name, %2$s: Follower name */
+		/* translators: 1: Blog name, 2: Follower name */
 		$subject = \sprintf( \__( '[%1$s] Follower: %2$s', 'activitypub' ), get_option( 'blogname' ), $actor['name'] );
-		/* translators: %1$s: Blog name, %2$s: Follower name */
-		$message = \sprintf( \__( 'New follower: %2$s', 'activitypub' ), get_option( 'blogname' ), $actor['name'] ) . "\r\n";
+		/* translators: 1: Blog name, 2: Follower name */
+		$message = \sprintf( \__( 'New Follower: %2$s', 'activitypub' ), get_option( 'blogname' ), $actor['name'] ) . "\r\n\r\n";
 		/* translators: %s: Follower URL */
 		$message .= \sprintf( \__( 'URL: %s', 'activitypub' ), $actor['url'] ) . "\r\n\r\n";
```

Comment:
```suggestion
		$message .= \sprintf( \__( 'URL: %s', 'activitypub' ), esc_url( $actor['url'] ) ) . "\r\n\r\n";
```

Reply:
Everything in `$message` should be escaped, actually. I didn't check that earlier.